test/k8s/updates.go

```diff
@@ -77,6 +78,20 @@ var _ = Describe("K8sUpdates", func() {
 
 		By("Waiting for pods to be terminated")
 		ExpectAllPodsTerminated(kubectl)
+
+		// Download the stable helm chart from GitHub
+		versionPath := filepath.Join(kubectl.BasePath(), "../old-charts/%s", helpers.CiliumStableVersion)
+		oldChartPath = filepath.Join(versionPath, fmt.Sprintf("cilium-%s/install/kubernetes/cilium", helpers.CiliumStableHelmChartVersion))
```

Comment:
nit: could we rename this to `stableChartPath` to make it more explicit what this is about?